config/e2e.yml

```diff
@@ -0,0 +1,28 @@
+boolTestKey: true
+numberTestKey: 1
+stringTestKey: 'string'
+
+rollbar:
+  accessToken: 'ROLLBAR_ACCESS_TOKEN'
+  env: 'ROLLBAR_ENV'
+
+jwt:
+  token: 'JWT_TOKEN'
+
+communication:
+  services: ['twilio', 'sendgrid']
+
+twilio:
+  verify:
+    accountSid: 'AC.TWILIO_ACCOUNT_SID'
+    authToken: 'TWILIO_AUTH_TOKEN'
+  messaging:
+    messagingServiceSid: 'TWILIO_MESSAGING_SERVICE_SID'
+
+sendgridApiKey: 'SG.SENDGRID_API_KEY'
```

Comment:
This should be removed as we are not using these integrations in e2e and thus, not requiring us to define these configuration values.

Reply:
Removed these already